platform/lang-impl/src/com/intellij/ide/util/PsiElementListCellRenderer.java

```diff
@@ -321,7 +321,7 @@ public void installSpeedSearch(PopupChooserBuilder<?> builder, boolean includeCo
   }
 
   public void installSpeedSearch(IPopupChooserBuilder builder) {
-    installSpeedSearch(builder, false);
```

Comment:
Please don't change the default behavior of this method. If you need `includeContainerText` to be true in a specific place, please make a change to that place.